external/vcm/tests/test_grid.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+import pytest
+from vcm.grid import get_grid
+
+
+@pytest.mark.parametrize("nx", [48, 24, 6])
+def test_grid_builder(nx):
+    lon, lat = get_grid(nx)
+    assert np.all(np.diff(lat[1, 0, :]) > 0)
+    assert np.all(np.diff(lon[1, :, 0]) > 0)
```

Comment:
Please add assertions that `lon.shape[1]`, `lon.shape[2]`, `lat.shape[1]`, and `lat.shape[2]` are all equal to `nx`.